Update add-service.md (#31)
edgelesssys/docs.edgeless.systems: marblerun/workflows/add-service.md

}
```

#### Graphene

To add an entry for your Graphene service, run the `graphene-sgx-get-token` tool on the `.sig` file you built in the previous step as follows. (`graphene-sgx-get-token` is installed with [Graphene](https://github.com/oscarlab/graphene/).)

```bash
graphene-sgx-get-token --sig hello.sig
```

> The tool's output will look like the following.

Reviewer: ```suggestion
The tool's output is similar to the following.
```
Also check the other 2 mentions.
Sounds a bit cleaner in my opinion.
Taken from the [google documentation style guide](https://developers.google.com/style/code-syntax).

erthost enclave.signed
```

`erthost` is the generic host for EdgelessRT Marbles, which will load your `enclave.signed`.

> For EGo (`ego run`), Graphene (`graphene-sgx`), and Occlum (`occlum run`) use their particular launch mechanism instead.

Reviewer: ```suggestion
For EGo (`ego marblerun`), Graphene (`graphene-sgx`), and Occlum (`occlum run`) use their particular launch mechanism instead.
```
